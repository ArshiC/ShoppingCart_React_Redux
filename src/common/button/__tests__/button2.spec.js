import React from "react";
import Button from "../index";
import renderer from "react-test-renderer";

describe("Button Component Snapshot Testing",()=>{
    test("it should render Button",()=>{
        const component = renderer.create(<Button src="https://www.telstra.com.au/content/dam/tcom/personal/mobile-phones/product-catalogue/samsung-galaxy-s9/samsung-galaxy-s9-purple-front.png"
        alt="Product Name" name="Product Name" />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
    
    test("it should render Button with Button-Type",()=>{
        const component = renderer.create(<Button src="https://www.telstra.com.au/content/dam/tcom/personal/mobile-phones/product-catalogue/samsung-galaxy-s9/samsung-galaxy-s9-purple-front.png"
        alt="Product Name" name="Product Name" buttonType="danger" />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})
