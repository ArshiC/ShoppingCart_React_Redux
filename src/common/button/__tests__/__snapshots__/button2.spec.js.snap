// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Snapshot Testing it should render Button 1`] = `
<button
  className="btn btn-sm btn-undefined"
>
  Product Name
</button>
`;

exports[`Button Component Snapshot Testing it should render Button with Button-Type 1`] = `
<button
  className="btn btn-sm btn-danger"
>
  Product Name
</button>
`;

exports[`it should render Button 1`] = `
<button
  className="btn btn-sm btn-undefined"
>
  Product Name
</button>
`;

exports[`it should render Button-Type 1`] = `
<button
  className="btn btn-sm btn-danger"
>
  Product Name
</button>
`;
